components/collection/voteDrop/DropContainer.vue

```diff
@@ -252,6 +253,14 @@ const { data: collectionData, refetch } = useGraphql({
   },
 })
 
+watch(collectionData, async () => {
+  if (collectionData.value) {
+    imageList.value = [
+      await sanitizeIpfsUrl(collectionData.value?.collectionEntity.image),
```

Comment:
you dont need await here as sanitize is a sync fn

Reply:
Updated